docs/reference/ccr/apis/follow/post-unfollow.asciidoc

```diff
@@ -54,6 +54,13 @@ POST /<follower_index>/_ccr/unfollow
 
 `follower_index` (required)::
   (string) the name of the follower index
+  
+==== Authorization
+
+If the {es} {security-features} are enabled, you must have `manage_follow_index` 
```

Comment:
only `manage_follow_index ` is needed in the follower cluster.

Reply:
Done, thanks!